app/config.py

```diff
@@ -13,23 +13,18 @@
 
 
 
-DATABASE_STAGE_URL = os.getenv('DATABASE_STAGE_URL')
-DATABASE_TEST_URL = os.getenv('DATABASE_TEST_URL')
-
-ENVIRONMENT = os.getenv('ENVIRONMENT', 'dev')
-TESTING = os.getenv("TESTING", 0)
+DATABASE_URL = os.getenv('DATABASE_URL')
+DATABASE_URL_TEST = os.getenv('DATABASE_URL_TEST')
+ENVIRONMENT = os.getenv('ENVIRONMENT')
+TESTING = os.getenv("TESTING")
```

Comment:
I don't see those env variables in README.md manual 🤔

Reply:
You shouldn't see DATABASE_URL and DATABASE_URL_TEST because there are created dynamically from docker compose
I missed ENVIRONMENT var I'm gonna add this to manual.